html/semantics/embedded-content/media-elements/track/track-element/cors/support/cors-tester.py

```diff
@@ -29,11 +28,10 @@ def main(request, response):
         response.headers.set('Access-Control-Allow-Origin', request.GET['origin'])
         response.headers.set('Access-Control-Allow-Credentials', 'true')
 
-    cors = request.headers.get("origin", "no")
-
+    cors = request.headers.get("origin", b"no")
     cookie = request.cookies.first(id, None)
     cookie_value = cookie.value if cookie is not None else "no"
-    line = 'cors = ' + cors + ' | cookie = ' + cookie_value;
+    line = 'cors = ' + cors.decode("utf-8") + ' | cookie = ' + cookie_value
```

Comment:
Is this casting a byte sequence to a string, basically? Are there really UTF-8 bytes in there?